src/test/java/de/uhd/ifi/se/decision/management/jira/rest/configrest/TestGetDecisionGroups.java

```diff
@@ -0,0 +1,79 @@
+package de.uhd.ifi.se.decision.management.jira.rest.configrest;
+
+import static org.junit.Assert.assertEquals;
+
+import javax.ws.rs.core.Response;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import de.uhd.ifi.se.decision.management.jira.TestSetUp;
+import de.uhd.ifi.se.decision.management.jira.model.DocumentationLocation;
+import de.uhd.ifi.se.decision.management.jira.model.KnowledgeElement;
+import de.uhd.ifi.se.decision.management.jira.model.KnowledgeStatus;
+import de.uhd.ifi.se.decision.management.jira.model.KnowledgeType;
+import de.uhd.ifi.se.decision.management.jira.model.impl.KnowledgeElementImpl;
+import de.uhd.ifi.se.decision.management.jira.persistence.DecisionGroupManager;
+import de.uhd.ifi.se.decision.management.jira.rest.ConfigRest;
+import de.uhd.ifi.se.decision.management.jira.rest.impl.ConfigRestImpl;
+
+public class TestGetDecisionGroups extends TestSetUp {
+
+    protected ConfigRest configRest;
+    private long id;
+    private String summary;
+    private String description;
+    private KnowledgeType type;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'type' could be replaced by a local variable.](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5160185)